tcms/testplans/templates/testplans/get.html

```diff
@@ -76,5 +76,57 @@ <h2 class="card-pf-title">
             </div>
         </div>
     </div>
+        <div class="row row-cards-pf">
+            <div class="col-xs-12 col-sm-12 col-md-12">
+                <div class="card-pf card-pf-accented">
+                    <div id="confirmed-testcases" class="list-group list-view-pf list-view-pf-view">
+                    </div>
+                </div>
+            </div>
+        </div>
 </div>
+
+<template id="no_test_cases">
+    <div class="list-group-item">
+        <div class="list-group-item-header">
+            <div class="list-view-pf-main-info">{% trans 'No testcases' %}</div>
+        </div>
+    </div>
+</template>
+
+<template id="test_case_row">
+    <div class="list-group-item">
+        <div class="list-group-item-header">
+            <div class="list-view-pf-expand">
+                <span class="fa fa-angle-right"></span>
+            </div>
+            <div class="list-view-pf-checkbox">
+                <input type="checkbox"></div>
+            <div class="list-view-pf-actions">
+                <div class="dropdown pull-right dropdown-kebab-pf">
+                    <button class="btn btn-link dropdown-toggle" type="button" id="dropdownKebabRight9"
+                            data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
+                        <span class="fa fa-ellipsis-v"></span>
+                    </button>
+                    <ul class="dropdown-menu dropdown-menu-right" aria-labelledby="dropdownKebabRight9">
+                        <li><a href="#">{% trans 'Change status' %}</a></li>
+                    </ul>
+                </div>
+            </div>
+            <div class="list-view-pf-main-info"></div>
+        </div>
+        <div class="list-group-item-container container-fluid hidden">
+            <div class="close">
+                <span class="pficon pficon-close"></span>
+            </div>
+            <div class="row">
+                <div class="col-md-12">Some test case widgets</div>
```

Comment:
Render TC.summary here as well. You can add an additional class to these elements so they are easier to be selected via jQuery.